wrangler-core/src/main/java/io/cdap/wrangler/utils/ColumnConverter.java

```diff
@@ -317,7 +317,8 @@ public static Schema getSchemaForType(String type, Integer scale) throws Directi
     type = type.toUpperCase();
     if (type.equals(ColumnTypeNames.DECIMAL)) {
       // TODO make set-type support setting decimal precision
-      typeSchema = Schema.nullableOf(Schema.decimalOf(38, scale));
+      scale = scale != null ? scale : 38;
+      typeSchema = Schema.nullableOf(Schema.decimalOf(76, scale));
```

Comment:
I think we can keep precision as 77 because according to the docs, a max precision of 76.76 is allowed. Besides, the other plugins (like Oracle) follow 38, 77 as default scale, precision.

Reply:
Done